src/include/stir/recon_buildblock/CUDA/CudaRelativeDifferencePrior.h

```diff
@@ -0,0 +1,36 @@
+#ifndef __stir_recon_buildblock_CudaRelativeDifferencePrior_h__
+#define __stir_recon_buildblock_CudaRelativeDifferencePrior_h__
+
+#include "stir/recon_buildblock/RelativeDifferencePrior.h"
+#include "stir/DiscretisedDensity.h"
+#include <cuda_runtime.h>
```

Comment:
move to cxx

Reply:
this is done but dim3 is cuda specific, I changed it for a native struct and removed the cuda_runtime include, aren't the other two needed?